File: app/src/main/java/com/slapshotapps/dragonshockey/dialogs/StatSortSelection.java
package com.slapshotapps.dragonshockey.dialogs;

public enum StatSortSelection {
    GamesPlayed("Games Played", 0), Goals("Goals", 1), Assists("Assists", 2), Points("Points", 3),
    PenaltyMinutes("Penalty Minutes", 4);

    String name;
    int index;

    StatSortSelection(String name, int index) {
        this.index = index;
        this.name = name;
    }

    public int getIndex() {
        return index;
    }

    public String getName() {
        return name;
    }
}

File: app/src/main/java/com/slapshotapps/dragonshockey/dialogs/StatSortSelection.kt
package com.slapshotapps.dragonshockey.dialogs

enum class StatSortSelection constructor(val statName: String, val index: Int) {
    GamesPlayed("Games Played", 0),
    Goals("Goals", 1),
    Assists("Assists", 2),
    Points("Points", 3),
    PenaltyMinutes("Penalty Minutes", 4);
}